utils.py

```diff
@@ -16,6 +16,12 @@ def _get_random_age(rng):
 		age = round(float(draw))
 	return age
 
+def _get_random_area(rng, location_type, num, total_area):
+	perc_dist = {"store":0.15, "misc":0.15, "workplace":0.2, "household":0.3, "park":0.5}
+	area = rng.dirichlet(np.ones(num))*(perc_dist[location_type]*total_area/2)
```

Comment:
- It will be nice to have an explanation of why you think Dirichlet distribution makes sense here.
- why do we need to divide the total_area by 2? I might be missing something here

Reply:
Can you also pass `rng` as the last argument?

Reply:
I have added the reasoning for using Dirichlet and using area/2 in the recent commit.

Reply:
Also 

> Can you also pass `rng` as the last argument?

Done